controllers/componentdetectionquery_controller.go

```diff
@@ -179,7 +179,7 @@ func (r *ComponentDetectionQueryReconciler) Reconcile(ctx context.Context, req c
 					r.SetCompleteConditionAndUpdateCR(ctx, req, &componentDetectionQuery, copiedCDQ, err)
 					return ctrl.Result{}, nil
 				}
-				log.Info(fmt.Sprintf("Look for devfile or dockerfile at the URL %s... %v", gitURL, req.NamespacedName))
+				log.Info(fmt.Sprintf("Look for devfile, dockerfile or containerfile at the URL %s... %v", gitURL, req.NamespacedName))
```

Comment:
small nit, is dockerfile/containerfile supposed to be capitalized?  If so, let's update it for consistency